schemas/1.0/application.json

```diff
@@ -5,32 +5,155 @@
   "description": "Attributes of a criminal legal aid application",
   "type": "object",
   "properties": {
-    "id": { "type": "string" },
-    "parent_id": { "type": ["string", "null"] },
-    "schema_version": { "type": "number" },
-    "reference": { "type": "integer" },
-    "application_type": { "type": "string", "enum": ["initial", "post_submission_evidence", "change_in_financial_circumstances"] },
-    "created_at": { "type": "string", "format": "date-time" },
-    "submitted_at": { "type": "string", "format": "date-time" },
-    "date_stamp": { "type": ["string", "null"], "format": "date-time" },
+    "id": {
```

Comment:
Most of these changes are formatting. I've highlighted the review_status change.